mix.exs

```diff
@@ -14,14 +14,19 @@ defmodule Statsig.MixProject do
 
   # Run "mix help compile.app" to learn about applications.
   def application do
-    [
-      mod: {Statsig.Application, []},
-      extra_applications: [:logger, :jason]
-    ]
+    case Mix.env() do
+      :test ->
+        [extra_applications: [:logger, :jason]]
+      _ ->
+        [
+          mod: {Statsig.Application, []},
+          extra_applications: [:logger, :jason]
+        ]
+    end
```

Comment:
I think you're trying to get the application not to start when you run tests. The easiest way to do this is to alias `test` to `test --no-start` in your aliases here. 

in the `application` function, return this:
```elixir
[ 
  mod: {Statsig.Application, []},
  extra_applications: [:logger, :jason]  # Do you need jason here? I don't think so
  aliases: aliases()
]
```

then, below, add an aliases function
```elixir
defp aliases() do 
  [test: "test --no-start"]
end
```